packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_governance_createcfp.test.ts

```diff
@@ -25,22 +26,34 @@ describe('createCfp', () => {
     builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic, RegTest)
 
     await testing.container.waitForWalletBalanceGTE(11)
-    await fundEllipticPair(testing.container, providers.ellipticPair, 12000100)
+    await fundEllipticPair(testing.container, providers.ellipticPair, fundAmount)
     await providers.setupMocks()
   })
 
   afterAll(async () => {
     await testing.container.stop()
   })
 
-  it('should createCfp', async () => {
+  it.each([{
+    nAmount: 999,
```

Comment:
suggestion: instead of using `nAmount` of `999`, `1000`, `1001`, i would test with larger variances (eg. `10`, `1000`, `100000`), especially since there is a change range, using test amounts of small variances might not be so obvious that it is working as intended (as the difference may fall within the change range)

Reply:
Fixed [`7e7f304`](https://github.com/JellyfishSDK/jellyfish/pull/1933/commits/7e7f304a501060667518639367c9b42e1407efc5)